main/keys.go

```diff
@@ -0,0 +1,84 @@
+	httpsKeyFileKey                 = "http-tls-key-file"
+	httpsCertFileKey                = "http-tls-cert-file"
+	apiAuthRequiredKey              = "api-auth-required"
+	apiAuthPasswordKey              = "api-auth-password" // #nosec G101
+	bootstrapIPsKey                 = "bootstrap-ips"
+	bootstrapIDsKey                 = "bootstrap-ids"
+	stakingPortKey                  = "staking-port"
+	stakingEnabledKey               = "staking-enabled"
+	p2pTLSEnabledKey                = "p2p-tls-enabled"
+	stakingKeyPathKey               = "staking-tls-key-file"
+	stakingCertPathKey              = "staking-tls-cert-file"
+	stakingDisabledWeightKey        = "staking-disabled-weight"
+	maxNonStakerPendingMsgsKey      = "max-non-staker-pending-msgs"
+	stakerMsgReservedKey            = "staker-msg-reserved"
+	stakerCPUReservedKey            = "staker-cpu-reserved"
+	networkInitialTimeoutKey        = "network-initial-timeout"
+	networkMinimumTimeoutKey        = "network-minimum-timeout"
+	networkMaximumTimeoutKey        = "network-maximum-timeout"
+	networkTimeoutIncreaseKey       = "network-timeout-increase"
+	networkTimeoutReductionKey      = "network-timeout-reduction"
+	benchlistFailThresholdKey       = "benchlist-fail-threshold"
+	benchlistPeerSummaryEnabledKey  = "benchlist-peer-summary-enabled"
+	benchlistDurationKey            = "benchlist-duration"
+	benchlistMinFailingDurationKey  = "benchlist-min-failing-duration"
+	pluginDirKey                    = "plugin-dir"
+	logsDirKey                      = "log-dir"
+	logLevelKey                     = "log-level"
+	logDisplayLevelKey              = "log-display-level"
+	logDisplayHighlightKey          = "log-display-highlight"
+	snowSampleSizeKey               = "snow-sample-size"
+	snowQuorumSizeKey               = "snow-quorum-size"
+	snowVirtuousCommitThresholdKey  = "snow-virtuous-commit-threshold"
+	snowRogueCommitThresholdKey     = "snow-rogue-commit-threshold"
+	snowAvalancheNumParentsKey      = "snow-avalanche-num-parents"
+	snowAvalancheBatchSizeKey       = "snow-avalanche-batch-size"
+	snowConcurrentRepollsKey        = "snow-concurrent-repolls"
+	whitelistedSubnetsKey           = "whitelisted-subnets"
+	adminAPIEnabledKey              = "api-admin-enabled"
+	infoAPIEnabledKey               = "api=info-enabled"
+	keystoreAPIEnabledKey           = "api-keystore-enabled"
+	metricsAPIEnabledKey            = "api-metrics-enabled"
+	healthAPIEnabledKey             = "api-health-enabled"
+	ipcAPIEnabledKey                = "api-ipcs-enabled"
+	xputServerPortKey               = "xput-server-port"
+	xputServerEnabledKey            = "xput-server-enabled"
+	ipcsChainIDsKey                 = "ipcs-chain-ids"
+	ipcsPathKey                     = "ipcs-path"
+	consensusGossipFrequencyKey     = "consensus-gossip-frequency"
+	consensusShutdownTimeoutKey     = "consensus-shutdown-timeout"
+	fdLimitKey                      = "fd-limit"
+	corethConfigKey                 = "coreth-config"
```

Comment:
I'm fine with leaving this as is for now... But the CLI args should work in a general way for all chains. Rather than having a config per VM (with the name of the VM hardcoded into avalanchego)... but as a stopgap I'm fine.